src/components/collapse/Collapse.tsx

```diff
@@ -43,23 +48,40 @@ export const Collapse = ({
     }
   }, [isExpand, setHeight, sizes.height, hideOverflow]);
 
+  useEffect(() => {
+    const { current } = el;
```

Comment:
The initial value for this `el` ref is `null`. Does it still allow you to spread `el` even that it's possibly null?

Reply:
Yes, because the `current` is the typed value, not the el. The passed generic type only affects what it holds.